### ISSUE TITLE
[Solved] RDP SetProps failed! Value does not fall within the expected range.

### ISSUE THREAD

**Issue by carrtohu** [mRemoteNG/mRemoteNG#1399]
## Discription of the Problem
I installed the latest version of mRemoreNG in my laptop and added a new RDP connection. But when I double clicked on the conncetion in the connection toolbar, the gui flashed and nothing happened. Log said "ERROR - RDP SetProps failed! Value does not fall within the expected range." and shows the error happened in RdpProtocol.cs:Line129. Connection succeed when I directly used the textbox in the menubar. Hope to find someway to solve the problem. Thanks.
## Recent Log
> 2019-04-09 21:55:21,520 [1] INFO  - Saving connections...
2019-04-09 21:55:21,567 [1] INFO  - Successfully saved connections
2019-04-09 21:55:35,285 [1] INFO  - mRemoteNG 1.76.19.35762 starting.
2019-04-09 21:55:35,301 [1] INFO  - Command Line: C:\Program Files (x86)\mRemoteNG\mRemoteNG.exe
2019-04-09 21:55:37,129 [1] INFO  - Microsoft Windows 10 专业版  64-bit
2019-04-09 21:55:37,129 [1] INFO  - Microsoft .NET CLR 4.0.30319.42000
2019-04-09 21:55:37,129 [1] INFO  - System Culture: zh-CN/中文(中国)
2019-04-09 21:55:37,129 [1] INFO  - Checking FIPS Policy...
2019-04-09 21:55:37,144 [1] INFO  - Checking Lenovo AutoScroll Utility...
2019-04-09 21:55:37,300 [1] INFO  - Override Culture: en/English
2019-04-09 21:55:37,425 [1] INFO  - Loading External Apps from: C:\Users\xxx\AppData\Roaming\mRemoteNG\extApps.xml
2019-04-09 21:55:38,347 [1] WARN  - PuttySessions.Watcher.StartWatching() failed.
找不到 
2019-04-09 21:55:38,363 [1] WARN  - XmingPortablePuttySessions.Watcher.StartWatching() failed: 'sessions' does not exist.
2019-04-09 21:55:41,113 [1] ERROR - RDP SetProps failed!
Value does not fall within the expected range.
   at MSTSCLib.IMsRdpClient8.set_Server(String pServer)
   at mRemoteNG.Connection.Protocol.RDP.RdpProtocol.Initialize() in C:\jenkins\workspace\Publish\Build mRemoteNG for Release\mRemoteV1\Connection\Protocol\RDP\RdpProtocol.cs:line 129
2019-04-09 21:55:41,128 [6] INFO  - Connection Event Closed
2019-04-09 21:55:41,128 [6] INFO  - Connection to xxx.xxx.xxx:xxx via RDP closed by user xxx.
2019-04-09 21:56:09,765 [1] INFO  - Protocol Event Connected
2019-04-09 21:56:09,765 [1] INFO  - Connection to "http://bugs.mremoteng.org/" via "HTTP" established by user "xxx" (Description: ""; User Field: "")
## Environment
Lenovo K2450 Laptop, I5-4200U 4GB 
Windows 10 1809 (17763.379) x64
mRemoteNG 1.76.19.35762 x86

**Comment by sparerd** [comment]
In your connection, do you have any port information in the hostname field?

**Comment by carrtohu** [comment]
> 
> 
> In your connection, do you have any port information in the hostname field?

Thanks sparerd. The full text is my.domain.name:9833. The same thing happened when I changed another Hostname/IP. MSTSC.exe worked normally to the same server in this laptop, and mRemoteNG worked normally to the same server in my other computers.

Here's the Log when I changed another IP:

> 2019-04-10 06:51:39,947 [1] INFO  - Saving connections...
2019-04-10 06:51:40,010 [1] INFO  - Successfully saved connections
2019-04-10 06:51:43,822 [1] INFO  - mRemoteNG 1.76.19.35762 starting.
2019-04-10 06:51:43,838 [1] INFO  - Command Line: C:\Program Files (x86)\mRemoteNG\mRemoteNG.exe
2019-04-10 06:51:45,681 [1] INFO  - Microsoft Windows 10 专业版  64-bit
2019-04-10 06:51:45,681 [1] INFO  - Microsoft .NET CLR 4.0.30319.42000
2019-04-10 06:51:45,681 [1] INFO  - System Culture: zh-CN/中文(中国)
2019-04-10 06:51:45,681 [1] INFO  - Checking FIPS Policy...
2019-04-10 06:51:45,697 [1] INFO  - Checking Lenovo AutoScroll Utility...
2019-04-10 06:51:45,759 [1] INFO  - Override Culture: en/English
2019-04-10 06:51:45,869 [1] INFO  - Loading External Apps from: C:\Users\xxx\AppData\Roaming\mRemoteNG\extApps.xml
2019-04-10 06:51:46,634 [1] WARN  - PuttySessions.Watcher.StartWatching() failed.
找不到 
2019-04-10 06:51:46,634 [1] WARN  - XmingPortablePuttySessions.Watcher.StartWatching() failed: 'sessions' does not exist.
2019-04-10 06:52:14,117 [1] ERROR - RDP SetProps failed!
Value does not fall within the expected range.
   at MSTSCLib.IMsRdpClient8.set_Server(String pServer)
   at mRemoteNG.Connection.Protocol.RDP.RdpProtocol.Initialize() in C:\jenkins\workspace\Publish\Build mRemoteNG for Release\mRemoteV1\Connection\Protocol\RDP\RdpProtocol.cs:line 129
2019-04-10 06:52:14,148 [4] INFO  - Connection Event Closed
2019-04-10 06:52:14,148 [4] INFO  - Connection to 192.168.1.1:3389 via RDP closed by user xxx.
